frontend/src/views/c-summary.vue

```diff
@@ -92,7 +94,10 @@
           v-bind:href="getReportIssueEmailLink(errorBlock.errorMessage)"
         )
           span {{ getReportIssueEmailAddress() }}
-      .error-message-box__failed-repo--reason(v-else) {{ errorBlock.errorMessage }}
+      .error-message-box__failed-repo--reason(v-else) {{ errorBlock.errorMessage }}\
+    .error-message-box__show-more-container(v-if="Object.keys(errorMessages).length >= 4")
```

Comment:
This should not contain `=` as by default 4 errors are shown according to line 80. Else, we could also change it to show only 3 errors by default.

```suggestion
    .error-message-box__show-more-container(v-if="Object.keys(errorMessages).length > 4")
```